### PR TITLE
feat: or-1111 cjm beheerder can do algemeen beheer

### PR DESCRIPTION
**Pull request by W0dan**

### CODE REVIEW

File: src/OrganisationRegistry.Api/Backoffice/Admin/Configuration/ConfigurationCommandController.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿namespace OrganisationRegistry.Api.Backoffice.Admin.Configuration;
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Admin/Configuration/ConfigurationCommandController.cs:1]: I believe we came to the conclusion this morning, not to split controllers when we don't actually send commands.

File: src/OrganisationRegistry.Api/Backoffice/Admin/Projections/ProjectionsCommandController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+namespace OrganisationRegistry.Api.Backoffice.Admin.Projections;
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Admin/Projections/ProjectionsCommandController.cs:1]: Same wrt not splitting.

File: src/OrganisationRegistry.Api/Backoffice/Admin/Task/TaskController.cs [hunk 3]
```diff
@@ -32,7 +32,7 @@ public TaskController(ICommandSender commandSender, ILogger<TaskController> logg
     /// <response code="200">If the task was performed.</response>
     /// <response code="400">If the task information does not pass validation.</response>
     [HttpPost]
-    [OrganisationRegistryAuthorize(Roles = Roles.AutomatedTask + "," + Roles.Developer)]
+    [OrganisationRegistryAuthorize(Role.AutomatedTask,Role.Developer)]
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Admin/Task/TaskController.cs:35]: space

File: src/OrganisationRegistry.Api/Backoffice/Body/Detail/BodyDetailCommandController.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    public BodyDetailCommandController(ICommandSender commandSender)
+        : base(commandSender)
+    {
+    }
+
+    /// <summary>Register a body.</summary>
+    /// <response code="201">If the body is registered, together with the location.</response>
+    /// <response code="400">If the body information does not pass validation.</response>
+    [HttpPost]
+    [OrganisationRegistryAuthorize]
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Body/Detail/BodyDetailCommandController.cs:28]: Move this to the class definition, just so we don't forget it?

File: src/OrganisationRegistry.Api/Backoffice/Kbo/KboRawController.cs [hunk 5]
```diff
@@ -32,7 +31,7 @@ public class KboRawController : OrganisationRegistryController
     /// <summary>Return raw result from magda kbo lookup.</summary>
     /// <response code="200">The raw request/response from magda kbo lookup.</response>
     [HttpGet("{kboNumberInput}")]
-    [OrganisationRegistryAuthorize(Roles = Roles.AlgemeenBeheerder + "," + Roles.Developer)]
+    [OrganisationRegistryAuthorize(Role.AlgemeenBeheerder,Role.Developer)]
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Kbo/KboRawController.cs:34]: space

File: src/OrganisationRegistry.Api/Backoffice/Organisation/BankAccount/OrganisationBankAccountCommandController.cs [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OrganisationRegistry.Infrastructure.Commands;
+using OrganisationRegistry.Organisation;
+
+[ApiVersion("1.0")]
+[AdvertiseApiVersions("1.0")]
+[OrganisationRegistryRoute("organisations/{organisationId}/bankAccounts")]
+[OrganisationRegistryAuthorize]
+public class OrganisationBankAccountCommandController : OrganisationRegistryController
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Backoffice/Organisation/BankAccount/OrganisationBankAccountCommandController.cs:16]: Just an idea: would it make sense to make 2 types of `OrganisationRegistryController`? One for reads, without `CommandSender`, the other for commands, with `CommandSender` and `OrganisationRegistryAuthorize` on it by default? @QuintenGreenstack @W0dan

**Review by QuintenGreenstack — APPROVED** [review]

**Review by QuintenGreenstack — APPROVED** [review]

**Pull request merged**